spring-cloud-data-admin/src/main/java/org/springframework/cloud/data/admin/AdminApplication.java

```diff
@@ -17,12 +17,18 @@
 package org.springframework.cloud.data.admin;
 
 import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.yarn.boot.YarnAppmasterAutoConfiguration;
+import org.springframework.yarn.boot.YarnClientAutoConfiguration;
+import org.springframework.yarn.boot.YarnContainerAutoConfiguration;
 
 /**
  * @author Mark Fisher
  */
 @SpringBootApplication
+@EnableAutoConfiguration(exclude = { YarnClientAutoConfiguration.class, YarnAppmasterAutoConfiguration.class,
```

Comment:
Do these configuration classes have some kind of conditions to enable auto configuration themselves (like `@ConditionOnProperty` on a profile? I am afraid we will start having a bunch of excludes this way. Is there other option to disable auto configuration of these?

Also, what happens when you run the admin on Yarn? should these auto-configurations need to be excluded there as well?